test/k8sT/Services.go

```diff
@@ -2232,15 +2248,10 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 
 		// Run on net-next and 4.19 but not on old versions, because of
 		// LRU requirement.
-		SkipItIf(helpers.DoesNotRunOn419OrLaterKernel, "Supports IPv4 fragments", func() {
-			options := map[string]string{}
-			// On GKE we need to disable endpoint routes as fragment tracking
-			// isn't compatible with that options. See #15958.
-			if helpers.RunsOnGKE() {
-				options["gke.enabled"] = "false"
-				options["tunnel"] = "disabled"
-			}
-			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, options)
+		SkipItIf(func() bool {
+			return helpers.DoesNotRunOnNetNextOr419Kernel() || helpers.RunsOnGKE()
+		}, "Supports IPv4 fragments", func() {
+			DeployCiliumAndDNS(kubectl, ciliumFilename)
```

Comment:
We should keep the old code here (i.e., run the test on GKE). This is mixed up because the PR disabling this test on GKE (https://github.com/cilium/cilium/pull/14639) is backported after the PR which fixed this test on GKE (https://github.com/cilium/cilium/pull/15959). Sorry about that :disappointed: